src/probnum/quad/kernel_embeddings/_matern_lebesgue.py

```diff
@@ -0,0 +1,222 @@
+"""Kernel embedding of Matern kernels with Lebesgue integration measure."""
+
+# pylint: disable=invalid-name
+
+from typing import Tuple, Union
+
+import numpy as np
+
+from probnum.quad._integration_measures import LebesgueMeasure
+from probnum.randprocs.kernels import Matern, ProductMatern
+
+
+def _kernel_mean_matern_lebesgue(
+    x: np.ndarray, kernel: Union[Matern, ProductMatern], measure: LebesgueMeasure
+) -> np.ndarray:
+    """Kernel mean of a ProductMatern or 1D Matern kernel w.r.t. its first argument
+    against a Lebesgue measure.
+
+    Parameters
```

Comment:
It would be great if we had the formulas for the kernel means and kernel variances in the docstrings as well similar to the formulas in the kernel classes. Currently the kernel embedding docs are not rendered in the API references but if we ever want to change that, it will be a pain to add the formulas later. What do you think?

Reply:
I guess this would be possible. The formulas for Matérns are rather hideous, so I'll see if they get rendered in a readable way. Alternatively, we could link to the kernel means PDF document that @alpiges and I have been preparing.

Reply:
Linking to a pdf sounds good as well. In case the pdf uses a different notation we can clarify in the docstring.

Reply:
Added the formulae in the docstrings even though these are not rendered in the API at the moment.